src/ContentCard/index.tsx

```diff
@@ -51,6 +51,10 @@ interface ContentCardProps {
    * Error state for `toggle` and `button` variants
    */
   error?: string;
+  /**
+   * Disables the card, preventing any click events.
+   */
+  disabled?: boolean;
```

Comment:
It's notable that design does have a concept of "disabled" for a ContentCard, but it's accomplished in code by composing ContentCard with a [DisabledShim](https://narmi.github.io/design_system/?path=/docs/components-disabledshim--docs)